minecraft_servers/pvpacademy/manifest.txt

```diff
@@ -0,0 +1,12 @@
+{
+  "server_name": "pvpacademy",
+  "nice_name": "PvPAcademy",
+  "direct_ip": "pvpacademy.net",
+
+  ],

```

Comment:
The manifest has to be a json file. Moreover you cannot close an non existing array here.
```suggestion
```